contracts/PartyBid.sol

```diff
@@ -391,7 +390,7 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
      * @return _maxBid the maximum bid
      */
     function _getMaximumBid() internal view returns (uint256 _maxBid) {
-        _maxBid = totalContributedToParty - _getFee(totalContributedToParty);
+        _maxBid = totalContributedToParty * 100 / 105;
```

Comment:
nit but it's a bit strange that we use `FEE_PERCENT` elsewhere but we hard-code in the 5% fee here

Reply:
word - https://github.com/PartyDAO/partybid/pull/40/commits/c26000845c70c1ad185041f32531b1e8b18a1525